documentation/examples/prometheus-kubernetes.yml

```diff
@@ -90,3 +90,55 @@ scrape_configs:
     regex: (.+)
     target_label: kubernetes_name
     replacement: $1
+
+# Example scrape config for probing services via the Blackbox Exporter.
+#
+# The relabeling allows the actual service scrape endpoint to be configured
+# via the following annotations:
+#
+# * `prometheus.io/probe`: Only probe services that have a value of `true`
+- job_name: 'kubernetes-services'
+
+  metrics_path: /probe
+  params:
+    module: [http_2xx]
+
+  kubernetes_sd_configs:
+  - api_servers:
+    - 'https://kubernetes.default.svc'
+    in_cluster: true
+
+  relabel_configs:
+  - source_labels: [__meta_kubernetes_role, __meta_kubernetes_service_annotation_prometheus_io_probe]
+    action: keep
+    regex: service;true
+  - source_labels: []
+    regex: .*
+    target_label: __address__
+    replacement: prom-blackbox.default.svc:9115
+  - source_labels: [__address__]
+    regex: (.*)(:80)?
+    target_label: __param_target
+    replacement: ${1}
+  - source_labels: [__param_target]
+    regex: (.*)
```

Comment:
You can remove this and the `replacement: ${1}` lines, these are the defaults now